repository_service_tuf/helpers/api_client.py

```diff
@@ -69,6 +74,94 @@ def is_logged(server: str, token: str):
             return Login(state=True, data=data)
 
     else:
-        click.ClickException(
-            f"Error {response.status_code} {response.json()['detail']}"
+        raise click.ClickException(
```

Comment:
I believe `is_logged` does not work as expected in all cases.
A `GET api/v1/token/` request may also return a `200` response with a body of:
```
{
  "detail": {
    "error": "Failed to validate token"
  }
}
```
See [here](https://github.com/vmware/repository-service-tuf-api/blob/main/repository_service_tuf_api/token.py#L210). In this case, `data` is `None` and ` data.get("expired")` will throw an `AttributeError`

I spotted it here, but if you validate it I can open a bug. (BTW quick fix: `data = response.json().get("data", {})`)

Reply:
> I spotted it here, but if you validate it I can open a bug. (BTW quick fix: `data = response.json().get("data", {})`)

That's a good spot. I'm implementing the quick fix.

> On the other hand, we are always validating a token with the same token in the headers, so I assume if the token is invalid it won't be because it can't be decoded... so it might be fine this way. I will not delete it, but I believe you can ignore my comment.

Maybe we can have a specific issue for this part to implement or remove this check.
Can you file the issue giving the background?

Reply:
@KAUTH when you open it could please link it here for future reference?

Reply:
I opened this issue
- https://github.com/vmware/repository-service-tuf-cli/issues/198